spot_api.yaml

```diff
@@ -393,19 +393,26 @@ paths:
               schema:
                 type: array
                 items:
-                  example:
-                    - 1499040000000
-                    - "0.01634790"
-                    - "0.80000000"
-                    - "0.01575800"
-                    - "0.01577100"
-                    - "148976.11427815"
-                    - 1499644799999
-                    - "2434.19055334"
-                    - 308
-                    - "1756.87402397"
-                    - "28.46694368"
-                    - "17928899.62484339"
+                  type: array 
```

Comment:
No need to re-add `type: array`

Reply:
It is an array of tuples, i.e. array of arrays. Documented [here](https://binance-docs.github.io/apidocs/spot/en/#kline-candlestick-data).

Reply:
Ok, the actual correction is the "example" section should have same indention as "items" in line 397. Otherwise, we'll get 3 levels of arrays in the UI.